src/Nethermind/Nethermind.AccountAbstraction/Data/UserOperation.Rlp.cs

```diff
@@ -0,0 +1,73 @@
+//  Copyright (c) 2021 Demerzel Solutions Limited
+//  This file is part of the Nethermind library.
+// 
+//  The Nethermind library is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+// 
+//  The Nethermind library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU Lesser General Public License for more details.
+// 
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+// 
+
+using Nethermind.Core.Crypto;
+using Nethermind.Serialization.Rlp;
+using Nethermind.Serialization.Rlp.Eip2930;
+
+namespace Nethermind.AccountAbstraction.Data
+{
+    public partial class UserOperation
```

Comment:
@marcindsobczak can we remove this?

Reply:
It was actually first, I created UserOperationDecoder, because this one wasn't compatible with IRlpValueDecoder and IRlpStreamDecoder. We can use UserOperationDecoder here now and delete UserOperation.Rlp. I will commit it within few minutes.